menu/scene_tabs.go

```diff
@@ -63,8 +64,8 @@ func buildTabs() Scene {
 		subLabel: "Scan your collection",
 		icon:     "add",
 		callbackOK: func() {
-			usr, _ := user.Current()
-			menu.Push(buildExplorer(usr.HomeDir, nil,
+			//usr, _ := user.Current()
+			menu.Push(buildExplorer(settings.Current.FileDirectory, nil,
```

Comment:
What motivates this change?

Reply:
Honestly I added the `FileDirectory` option because I was getting tired of having to navigate back to my ROM collection everytime I wanted to test out a change to the scanner. This could be reverted since that's not a typical use case, or per you comment on defaults.go could be changed back to the usr.HomeDir or something similar.